### ISSUE TITLE
Favicon(s) blocked by some adblockers

### ISSUE THREAD

**Issue by nsuchy** [freeCodeCamp/guide#8124]
I have noticed that the favicon(s) are being blocked by some adblockers. When we first implemented the favicon(s), we linked to freeCodeCamp's existing S3 Bucket (Ref: https://github.com/freeCodeCamp/guides/pull/118). What would be the pros and cons of hosting the favicons directly on the guides server? Should we adjust this to accommodate for those users or leave it as is?

**Comment by thecodingaviator** [comment]
@nsuchy , maybe you'd want to revive this thread?

**Comment by nsuchy** [comment]
@QuincyLarson You mentioned to me that the guide is getting merged into core. I am blocking this issue for now and we can investigate if the same issue occurs once guide is merged into core.
